crypto/keys/ed25519/ed25519.go

```diff
@@ -18,8 +18,8 @@ import (
 //-------------------------------------
 
 const (
-	PrivKeyName = "cosmos/PrivKeyEd25519"
-	PubKeyName  = "cosmos/PubKeyEd25519"
+	PrivKeyName = "tendermint/PrivKeyEd25519"
```

Comment:
had to revert this to support backwards compat.

Reply:
who is relaying on this?

Reply:
now that i think some more about it, it may not be needed, will test locally